pkg/services/accesscontrol/roles.go

```diff
@@ -56,8 +60,9 @@ var usersOrgEditRole = RoleDTO{
 }
 
 var usersAdminReadRole = RoleDTO{
-	Name:    usersAdminRead,
-	Version: 1,
+	Name:        usersAdminRead,
+	Version:     2,
+	Description: "Allows to list and get users and related information.",
```

Comment:
```suggestion
	Description: "Allows to list and get users and related information.",
```
Allows [what? / who?] to list...

Reply:
```suggestion
	Description: "Role to allow users to list users in a given organization, retrieve the teams they are part of, list their authentication tokens, get their connection quotas.",
```